amundsen_application/static/js/ducks/tableMetadata/api/v0.ts

```diff
@@ -104,17 +98,14 @@ export function metadataTableOwners(tableData: TableMetadata) {
 }
 
 /* TODO: Typing this method generates redux-saga related type errors that need more dedicated debugging */
-export function metadataUpdateTableOwner(action, tableData) {
+export function metadataUpdateTableOwner(action, tableData: TableMetadata) {
```

Comment:
nit: Can you add a `TODO` for putting the `key` on the action and using `action.key`?